platform/app/src/routes/Mode/Mode.tsx

```diff
@@ -43,6 +46,22 @@ function defaultRouteInit(
         SeriesInstanceUID
       );
 
+      // checks if the series filter was used, if it exists
+      const seriesInstanceUIDs = filters?.seriesInstanceUID;
+      if (
+        seriesInstanceUIDs?.length &&
+        !isSeriesFilterUsed(seriesMetadata.instances, filters) &&
+        !issuedWarningSeries.includes(seriesInstanceUIDs[0])
+      ) {
+        // stores the series instance filter so it shows only once the warning
+        issuedWarningSeries.push(seriesInstanceUIDs[0]);
+        uiNotificationService.show({
+          title: 'Series filter',
+          message: `Series ${seriesInstanceUIDs[0]} could not be found in the current study`,
```

Comment:
Considering this message will only display when the list specified in `seriesinstanceuid` has NO corresponding series in the study, then shouldn't we output the entire filter list (and not just the first entry) and say something instead like the following?

"Each of the series: ${seriesInstanceUIDs}  are not part of the current study. The entire study is being displayed"

Not sure of the wording here, but maybe somebody else can help with that too.

Reply:
I think its fine. I added 'in filter'